### PR TITLE
Added Support for CV_READ

### PR DESCRIPTION
**Pull request by littleyoda**
I've added Support for CV_READ.

### CODE REVIEW

**Comment by grizeldi** [comment]
Didn't see that coming, but looks cool! I'll test it when I have some time. Also please make sure that javadoc comments on every class you added are correct, so I can figure out how this is supposed to work easier.

**Comment by littleyoda** [comment]
I changed from Broadcasts to Response

**Comment by grizeldi** [comment]
I'll test this probably on Wednesday.
Didn't check the doc yet, what is CVNack? CV not found or what?
Thanks for all the effort so far ;)

File: src/z21Drive/Z21.java [hunk 1]
```diff
@@ -241,6 +261,17 @@ static Z21Response responseFromPacket(DatagramPacket packet){
...
+        else if (header1 == 0x40 && header2 == 0x00 && xHeader == 0x64 && (array[5] & 255) == 0x14)
+            return new Z21ResponseLanXCVResult(array);
+        else if (header1 == 0x40 && header2 == 0x00 && xHeader == 0x61 && (array[5] & 255) == 0x13)
+            return new Z21ResponseLanXCVNACK(array);
+        else {
+            Logger.getLogger("Z21 Receiver").warning("Received unknown response message. Array:");
+            for (byte b : array)
+                System.out.print("0x" + String.format("%02X ", b));
+            System.out.println();
+            System.out.println("     " + "     " + "H1   " + "H2   " + "xH   ");
```
Comment (grizeldi) [inline src/z21Drive/Z21.java:273]: Umm, what? Is this supposed to print the values?

**Comment by grizeldi** [comment]
Done. Thanks for your effort.
It made me fix some bugs which I would otherwise probably never find. Also I had to create a programming mode broadcast, otherwise it reported an unknown broadcast.

**Pull request merged**